tests/tests/fv3_ccpp_gfs_v15p2_debug_RRTMGP

```diff
@@ -0,0 +1,85 @@
+###############################################################################
```

Comment:
The name of this file should be `tests/tests/fv3_ccpp_gfs_v15p2_RRTMGP_debug` and not `tests/tests/fv3_ccpp_gfs_v15p2_debug_RRTMGP`?